deploy/template.yaml

```
@@ -292,6 +292,7 @@ Resources:
               IDENTITY_KIWI_TARGET: "/perfTest/identity/kiwi/target"
               IDENTITY_ORCH_STUB_URL: "/perfTest/identity/orchStubUrl"
               IDENTITY_PASSPORT_URL: "/perfTest/identity/lime/passportUrl"
+              IDENTITY_SPOT_SQS: "/perfTest/identity/spot/spotSQS"
```

```suggestion
              IDENTITY_SPOT_SQS: "/perfTest/identity/spot/SQS"
```
We can avoid repeating ourselves here

---

Updated